gobblin-modules/gobblin-sql/src/main/java/org/apache/gobblin/source/jdbc/JdbcExtractor.java

```diff
@@ -655,7 +655,9 @@ private void parseInputQuery(String query) {
     ResultSet resultSet = null;
     try {
       this.jdbcSource = createJdbcSource();
-      this.dataConnection = this.jdbcSource.getConnection();
+      if(this.dataConnection == null) {
```

Comment:
Please add a space after `if` here and below.

Reply:
Done.